klippy/mcu.py

```diff
@@ -64,19 +66,20 @@ def __init__(self, serial, msgformat, respformat, oid=None,
         if cmd_queue is None:
             cmd_queue = serial.get_default_command_queue()
         self._cmd_queue = cmd_queue
-    def _do_send(self, cmds, minclock, reqclock):
-        xh = self._xmit_helper(self._serial, self._response, self._oid)
+    def _do_send(self, cmds, minclock, reqclock, retry):
+        xh = self._xmit_helper(self._serial, self._response, self._oid, retry)
         reqclock = max(minclock, reqclock)
         try:
             return xh.get_response(cmds, self._cmd_queue, minclock, reqclock)
```

Comment:
Not a big deal either way, but at first glance it seems a little odd to pass `retry` to the constructor instead of as a parameter to `xh.get_response`.